report/docker_run.sh

```diff
@@ -85,6 +86,60 @@ then
   echo "DELAY was not specified as the sixth argument. Defaulting to ${DELAY:?}."
 fi
 
+launch_local_introspector_deployment() {
```

Comment:
This function is quite long and independent, would it be better to do this in a separate file like `upload_report.sh`?

Reply:
Done, thanks!